independent-projects/parent/pom.xml

```diff
@@ -623,11 +607,9 @@
                             <groupId>org.apache.maven.plugins</groupId>
                             <artifactId>maven-gpg-plugin</artifactId>
                             <configuration>
-                                <!-- Prevent gpg from using pinentry programs -->
-                                <gpgArguments>
-                                    <arg>--pinentry-mode</arg>
-                                    <arg>loopback</arg>
-                                </gpgArguments>
+                                <bestPractices>true</bestPractices>
+                                <useAgent>false</useAgent>
+                                <signer>bc</signer>
```

Comment:
Apparently, these are today's best practices so let's try it :).

Reply:
This will use BouncyCastle java signer, so no need to `gpg` native executable (nor signature installation, etc).